src/components/InboxButton/index.jsx

```diff
@@ -0,0 +1,9 @@
+function InboxButton (){
```

Comment:
See my comment on your Dropdown component - this InboxButton should be replaced with a single re-usable component that can be used for all nav links